src/GrpcProtos/capture.proto

```diff
@@ -107,6 +107,22 @@ message FunctionCall {
   repeated uint64 registers = 8;
 }
 
+// FunctionEntry and FunctionExit are emitted by user space instrumentation.
+message FunctionEntry {
+  uint32 pid = 1;
```

Comment:
Do we need the pid? I think I added it since it was present in the `FunctionCall` message but it is always the same number (see OrbitUserSpaceInstrumentation.cpp:108) and there is little use in sending it around, right? Same below in `FunctionExit`.

If you agree: Can you remove it from `FunctionCallEvent` as well?

Reply:
Well, what about multi-process capturing? I'd rather keep it. We also have that in almost every message.

Reply:
Hmm, I have not considered multi-process capturing. We could still figure out the pid from the tid. But if this adds complexity it might be fine to just send the pid around.

Reply:
Right, we have pid/tid mapping. If we have that available on the client, we should actually consider dropping pid everywhere.

Reply:
If the mapping is not saved with the capture that will cause issues when loading captures. 

The pid field seems to be everywhere. Let's keep it and discuss this as a potential optimization (not sure it's worth it).

Reply:
I see that the situation solved itself :D Yes, we have pid+tid everywhere so we do that here too. It would be possible to keep the association between tid and pid and use that (we mentioned it a few times), but it's much more work than the scope of this change.